src/EA.Weee.Web/Views/OrganisationRegistration/SearchResults.cshtml

```diff
@@ -83,7 +83,7 @@
         <p class="govuk-body">
             If you are a small producer of Electrical and Electronic Equipment (EEE) who hasn't previously been registered,
             you will need to create an account.
-            <a class="govuk-link govuk-link--no-visited-state" href="@Url.Action("TonnageType", "OrganisationRegistration", new {searchTerm = Model.SearchTerm})">Register for a small producer account</a>.
+            <a class="govuk-link govuk-link--no-visited-state" href="@Url.Action("RegisterSmallProducer", "OrganisationRegistration", new {searchTerm = Model.SearchTerm})">Register for a small producer account</a>.
```

Comment:
Isn't this going to skip the tonnage screen now?

Reply:
I added a slight redirect to this so that I can clear down any in progress transactions. It goes to this action and then redirects off as the previous view did.

![image](https://github.com/user-attachments/assets/4adbfc2f-fec8-4e3c-999b-89aa65215d85)